README.md

```diff
@@ -1,28 +1,52 @@
 # vosk-cli
 
-This python package serves as an Vosk interface for Opencast. It allows to generate subtitles (WebVVT files) from Video and Audio sources via Vosk.
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/elan-ev/vosk-cli/Build%20and%20publish)
+![GitHub](https://img.shields.io/github/license/elan-ev/vosk-cli)
+![PyPI](https://img.shields.io/pypi/v/vosk-cli)
+
+This python package serves as an Vosk interface for Opencast. It allows to generate subtitles (WebVTT files) from Video and Audio sources via Vosk.
 
 ## Installation
 
 ### 1. Install vosk-cli
+To install the [latest stable version of vosk-cli](https://pypi.org/project/vosk-cli/), run
+
+```
+pip install vosk-cli
+```
 
-Clone this project and run
+Alternatively, to install the latest development version, clone this project and inside the project directory run
 
 ```
-python setup.py install
+pip install .
 ```
 
 ### 2. Install dependencies
 
-- FFmpeg
-- `pip install -r requirements.txt`
+vosk-cli uses ffmpeg to preprocess input files. The easiest way to install ffmpeg is by using a package manager.
+To install ffmpeg on **Linux**, run
+
+```
+sudo apt install ffmpeg
+```
+On **MacOS**, run 
+```
+brew install ffmpeg
+```
+
+If you are on **Windows** and/or want or need to install from source, visit
+[FFmpeg.org](https://ffmpeg.org/download.html) and follow the instructions for your operating system.
 
 ### 3. Download the language model
 
-Go to `https://alphacephei.com/vosk/models` and download at least the English language model.
+Go to [https://alphacephei.com/vosk/models](https://alphacephei.com/vosk/models) and download at least the English language model.
+
+You can unzip the folder of the language model into any directory, but it is recommended to create and use a `./models` folder in the project directory and to rename your unzipped model to `eng` for example. Please use 3 digit language codes for the directory name. The default and fallback language directory of vosk-cli is `eng`. The directory name should match the workflow operation field `language-code`.
```

Comment:
```suggestion
You can unzip the folder of the language model into any directory, but it is recommended to create and use a `./models` folder in the project directory.
```

- I would leave Opencast configuration out of this
- I would discourage people from naming models `eng`, since you don't know any longer, which model this is. There are several different modelf for English, for example.